src/saga/login.js

```diff
@@ -46,7 +46,7 @@ import {
 
 import {
   downloadVmConsole,
-} from './consoles'
+} from './vm-console'
```

Comment:
Please use for names `camelCase`.

Reply:
I'll rename `src/saga/vm-console` to `src/saga/console` to be consistent with the names under `src/actions`

Reply:
Fixed the file names as well.